### PR TITLE
Feat: store draco_grid_size in info file

### PR DESCRIPTION
**Pull request by manuel-castro**
The draco_grid_size is necessary for mesh vertex deduplication and differs in each dataset

### CODE REVIEW

**Comment by william-silversmith** [comment]
Are the updates to the graph server live?

**Comment by manuel-castro** [comment]
Yessir

**Comment by william-silversmith** [comment]
Would you mind rebasing on master? I fixed the travis killing dependency issue.

**Comment by fcollman** [comment]
i am presently stitching draco meshes that don't have an upgraded graph server... is there a workaround for me to specify it manually?  can i override the cv.info after it downloads from the server?

**Comment by manuel-castro** [comment]
> i am presently stitching draco meshes that don't have an upgraded graph server... is there a workaround for me to specify it manually? can i override the cv.info after it downloads from the server?

Good question; yeah there is:
`
cv = CloudVolume('graphene://server_url)
cv.meta.mesh_metadata = {"uniform_draco_grid_size": 21}
`

**Comment by manuel-castro** [comment]
> Would you mind rebasing on master? I fixed the travis killing dependency issue.

I did, but now there's some blosc installation issue :dog:

Maybe pin blosc to 1.8.1 for now?

**Comment by william-silversmith** [comment]
Yea let's do that. All these dependencies breaking is such a headache...

**Comment by william-silversmith** [comment]
If I understand correctly, `uniform_draco_grid_size` means that there are no cross-level meshes to stitch?

File: cloudvolume/mesh.py [hunk 1]
```diff
@@ -23,17 +23,19 @@ def deprecation_notice(key):
...
+  # Draco rounds up
+  is_on_chunk_behind = np.any(
+    dist_to_chunk_behind < (draco_grid_size / 2),
+    axis=1,
+  )
+  is_on_chunk_ahead = np.any(
+    dist_to_chunk_ahead <= (draco_grid_size / 2),
+    axis=1,
+  )
+  return np.logical_or(is_on_chunk_behind, is_on_chunk_ahead)
```
[the remark this answers is not in the document]
Comment (manuel-castro) [reply cloudvolume/mesh.py]: It will return true only for vertices within half a draco_grid_size from a chunk border

**Comment by manuel-castro** [comment]
> If I understand correctly, `uniform_draco_grid_size` means that there are no cross-level meshes to stitch?

It means the grid size is the same at each level

File: cloudvolume/datasource/graphene/mesh.py [hunk 2]
```diff
@@ -104,11 +104,34 @@ def get(
...
+      if self.meta.chunks_start_at_voxel_offset:
+        offset = self.meta.voxel_offset(self.config.mip)
+      else:
+        offset = Vec(0,0,0)
 
       if remove_duplicate_vertices:
         mesh = mesh.consolidate()
       else:
-        mesh = mesh.deduplicate_chunk_boundaries(self.meta.mesh_chunk_size * resolution, is_draco=is_draco)
+        if is_draco:
```
[the remark this answers is not in the document]
Comment (manuel-castro) [reply cloudvolume/datasource/graphene/mesh.py]: Good point. I unnested it by one layer; one of the statements was unnecessarily nested

**Comment by william-silversmith** [comment]
`NameError: global name 'exceptions' is not defined` Huh, I thought this was in the imports. I guess not.

File: cloudvolume/mesh.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (william-silversmith) [reply cloudvolume/mesh.py]: How many draco grids to a chunk again (typically)?

**Comment by william-silversmith** [comment]
>     If I understand correctly, uniform_draco_grid_size means that there are no cross-level meshes to stitch?
> 
> It means the grid size is the same at each level

Would you mind adding some comments in the code about how the system operates? I think otherwise it might be difficult to maintain.

File: cloudvolume/mesh.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (manuel-castro) [reply cloudvolume/mesh.py]: For fly and pinky we used grid size = 21nm, so ~97.5 in x and y; ~975 in z

**Comment by manuel-castro** [comment]
> > ```
> > If I understand correctly, uniform_draco_grid_size means that there are no cross-level meshes to stitch?
> > ```
> > 
> > 
> > It means the grid size is the same at each level
> 
> Would you mind adding some comments in the code about how the system operates? ~I think otherwise it might be difficult to maintain.~ More like, I'll probably forget what everything means

Fo' shizzle.

**Pull request merged**